migrate/20231110_add_host_updating.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+Sequel.migration do
+  change do
+    add_enum_value(:allocation_state, "updating")
```

Comment:
Somehow, this doesn't strike me as an "allocation" state, perhaps we need a value for non-monotonic pauses that are reversed. `paused` seems suitable?